src/components/CourseTable/CourseTable.test.jsx

```diff
@@ -9,9 +9,28 @@ describe('CourseTable', () => {
     expect(component).toMatchSnapshot();
   });
 
-  it('hides table and button when not whitelisted', () => {
-    process.env.ORG_WHITELIST = ['fake'];
-    const component = shallow(<CourseTable />);
+  it('hides table and button when blacklisted', () => {
+    const publisherUserInfo = { organizations: [{ key: 'fake1', name: 'fake_name1' }] };
+    const component = shallow(<CourseTable publisherUserInfo={publisherUserInfo} />);
+    expect(component).toMatchSnapshot();
+  });
+
+  it('displays table and button when not blacklisted', () => {
+    const publisherUserInfo = { organizations: [{ key: 'fake2', name: 'fake_name2' }] };
+    const component = shallow(<CourseTable publisherUserInfo={publisherUserInfo} />);
+    expect(component).toMatchSnapshot();
+  });
+
+  it('displays table and button when no blacklist exists', () => {
+    process.env.ORG_BLACKLIST = [];
```

Comment:
I don't think this actually overwrites the value since it is set upon import. So this test probably isn't doing what we expect it to.

Reply:
ah ok thanks - I'll take a look